docs/docs/index.html

```diff
@@ -115,38 +115,38 @@ <h3 class="title"><a href="index.html">OpenShift Pipelines</a></h3>
 </div>
 <div class="ulist">
 <ul>
 <li>
-<p>OpenShift Pipelines are designed for microservices and decentralized teams</p>
+<p>OpenShift Pipelines is designed for microservices and decentralized teams</p>
 </li>
 <li>
-<p>They are standard CI/CD pipeline definitions that are easy to extend and integrate with existing tools, enabling you to scale on-demand</p>
+<p>OpenShift Pipelines uses standard CI/CD pipeline definitions that are easy to extend and integrate with existing Kubernetes tools, enabling you to scale on-demand</p>
 </li>
 <li>
-<p>You can use them to build images with Kubernetes tools such as Source-to-Image (S2I), Buildah, Buildpacks, and Kaniko and are portable across any Kubernetes platform</p>
+<p>You can use OpenShift Pipelines to build images with Kubernetes tools, such as Source-to-Image (S2I), Buildah, Buildpacks, and Kaniko</p>
 </li>
 <li>
-<p>You can deploy applications to multiple platforms such as Kubernetes, Serverless, and Virtual Machines (VMs) and are portable across any Kubernetes platform</p>
+<p>It can deploy applications to multiple platforms such as Kubernetes, Serverless, and Virtual Machines (VMs)</p>
 </li>
 <li>
-<p>You can use the OpenShift Developer Console to see your Pipelines and integrate with them.</p>
+<p>You can use the OpenShift Developer Console to see create Tekton resources, view logs of pipeline runs, and manage pipelines under OpenShift project namespaces</p>
 </li>
 </ul>
 </div>
 <h2 id="_additional_resources" class="discrete">Additional resources</h2>
 <div class="ulist">
 <ul>
 <li>
-<p>For more details on defining pipelines, see the <a href="https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/">Custom Resources documentation</a> .</p>
+<p>For more details on defining pipelines, see the <a href="https://openshift.github.io/pipelines-docs/docs/docs/assembly_using-pipelines.html">the Using Pipelines section</a></p>
 </li>
 <li>
-<p>For more information on Tekton see the <a href="https://github.com/tektoncd">Tekton Pipelines documentation</a>.</p>
+<p>For more information on Tekton see the <a href="https://github.com/tektoncd">Tekton Pipelines documentation</a></p>
 </li>
 <li>
-<p>Visit our <a href="https://github.com/openshift/pipelines-docs">GitHub</a> repository for more information.</p>
+<p>Visit our <a href="https://github.com/openshift/pipelines-docs">GitHub</a> repository to report any issues with the OpenShift Pipelines documentation</p>
```

Comment:
I would use this bullet point as an opportunity to ask for feedback on the docs instead of saying there is more information on GitHub.